src/main/java/com/gluonhq/substrate/target/DarwinTargetConfiguration.java

```diff
@@ -53,6 +53,7 @@ String getAdditionalSourceFileLocation() {
                 libPath + "libprism_es2.a", libPath + "libglass.a",
                 libPath + "libjavafx_font.a", libPath + "libjavafx_iio.a"));
         if (usePrismSW) {
+            answer.add(libPath + "libdecora_sse.a");
```

Comment:
why is this needed? Who calls it?

Reply:
In theory the native library is [loaded](https://github.com/openjdk/jfx/blob/22d4343fe8563c2931910b98e8f18c6fd4a48f05/modules/javafx.graphics/src/main/java/com/sun/scenario/effect/impl/sw/sse/SSERendererDelegate.java#L41) from `SSERendererDelegate`.

Reply:
However I'm not sure if it required (at least for the small number of effects we are using now), so probably this can be removed from here.